utils/warnings.mli

```diff
@@ -78,9 +78,24 @@ type t =
   | Constraint_on_gadt                      (* 62 *)
 ;;
 
+(* For warnings that can be enabled by attributes on external declarations,
+    the status indicates whether we should follow the current state of the
+    warning or the one given by those external declarations.
+
+    An example of such warning is Fragile_match, which can be (de)activated on a
+    per-type basis. See PR#7310 for an extended discussion.
+*)
```

Comment:
Yes there is also GPR but it's a different set of PRs. I was just talking about the naming of Mantis  Problem Reports.